server/monitor-types/rabbitmq.js

```diff
@@ -0,0 +1,68 @@
+const { MonitorType } = require("./monitor-type");
+const { log, UP, DOWN } = require("../../src/util");
+const { axiosAbortSignal } = require("../util-server");
+const axios = require("axios");
+
+class RabbitMqMonitorType extends MonitorType {
+    name = "rabbitmq";
+
+    /**
+     * @inheritdoc
+     */
+    async check(monitor, heartbeat, server) {
+        let baseUrls = [];
+        try {
+            baseUrls = JSON.parse(monitor.rabbitmqNodes);
+        } catch (error) {
+            throw new Error("Invalid RabbitMQ Nodes");
+        }
+
+        heartbeat.status = DOWN;
+        for (let baseUrl of baseUrls) {
+            try {
+                // Without a trailing slash, path in baseUrl will be removed. https://example.com/api -> https://example.com
+                if ( !baseUrl.endsWith("/") ) {
+                    baseUrl += "/";
+                }
+                const options = {
+                    // Do not start with slash, it will strip the trailing slash from baseUrl
+                    url: new URL("api/health/checks/alarms/", baseUrl).href,
+                    method: "get",
+                    timeout: monitor.timeout * 1000,
+                    headers: {
+                        "Accept": "application/json",
+                        "Authorization": "Basic " + Buffer.from(`${monitor.rabbitmqUsername || ""}:${monitor.rabbitmqPassword || ""}`).toString("base64"),
+                    },
+                    // Use axios signal to handle connection timeouts https://stackoverflow.com/a/74739938
```

Comment:
nit: I think this part was already clear, I was just a bit confused about `validateStatus`

```suggestion
```